### PR TITLE
Use the ES6 import method to import module

### PR DESCRIPTION
**Pull request by YuZhangYuanYYZ**

### CODE REVIEW

File: todoListAPI.js [hunk 1]
```diff
@@ -1,13 +1,57 @@
...
+      return todo
+    }
+  })
+  res.send(todo);
+})
+
+app.delete('/todos/:id',(req,res)=>{
+  const id = Number(req.params.id);
+  const newTodos = todos.filter(todo=>todo.id!==id);
+  todos = newTodos;
```
Comment (lanmengyujing) [inline todoListAPI.js:21]: this can be deleted now.

File: todoListAPI.js [hunk 2]
```diff
@@ -1,13 +1,57 @@
...
+  res.send({}) ;
```
Comment (lanmengyujing) [inline todoListAPI.js:22]: does this follow RESTful API response format?

File: todoListAPI.js [hunk 3]
```diff
@@ -1,13 +1,57 @@
...
+      const newTodo = {...todo,"content":newContent}
+      return newTodo;
+    }
+    return todo;
+  })
+  const resTodo = todos.find(todo => todo.id === id);
+  if (resTodo === []) {
+    res.status(404).send("出错了！");
+  } else {
+    res.send(resTodo);
```
Comment (lanmengyujing) [inline todoListAPI.js:39]: this PUT method also do not follow RESTfull pattern

File: todoData.js [hunk 4]
```diff
@@ -1 +1 @@
-export const todos =["hello one","hello two","hello three","hello four"];
\ No newline at end of file
+export let todos =[{"content":"hello zero","id":0},{"content":"hello one","id":1},{"content":"hello two","id":2},{"content":"hello three","id":3}];
```
Comment (lanmengyujing) [inline todoData.js:1]: where is the Store you have created?

**Review by lanmengyujing — CHANGES_REQUESTED** [review]
it is better to have a store to handle the update, delete, add, query for todos.
```
TodoStore {
addTodo()
deleteTodo()
updateTodo()
createTodo()
}
```

File: todoListAPI.js (hunk 2 above)
Comment (jingliu-rea) [reply todoListAPI.js:22]: empty does not mean {}

File: todoListAPI.js (hunk 3 above)
Comment (jingliu-rea) [reply todoListAPI.js:39]: resTodo is a list of todo, not a single object

File: todoListAPI.js (hunk 2 above)
Comment (jingliu-rea) [reply todoListAPI.js:22]: res.status(200).send()

Comment (lanmengyujing) [reply todoListAPI.js:22]: empty does not mean {}
```
res.status(200).send()
```

File: todoListAPI.js (hunk 3 above)
Comment (lanmengyujing) [reply todoListAPI.js:39]: resTodo is a list of todo, not a single object

File: todoListAPI.js (hunk 1 above)
Comment (YuZhangYuanYYZ) [reply todoListAPI.js:21]: ok.thank for your reminding.

File: todoListAPI.js (hunk 2 above)
Comment (YuZhangYuanYYZ) [reply todoListAPI.js:22]: yeah,you're right.

File: todoListAPI.js (hunk 3 above)
Comment (YuZhangYuanYYZ) [reply todoListAPI.js:39]: ???resTodo is a single object.

**Pull request merged**